src/shared_utilities/Domains.jl

```diff
@@ -1206,7 +1206,16 @@ function landsea_mask(
     return binary_mask
 end
 
-function landsea_mask(domain::Domains.AbstractDomain; kwargs...)
+# Points and Columns do not have a horizontal dim, so a horizontal mask cannot be applied
+landsea_mask(domain::Union{Point, Column}; kwargs...) = nothing
```

Comment:
Could you explain why this is needed? I think we only call the landsea_mask inside of `global_domain`? or is this inside the nanchecker? I think there is an option to run that without a mask